docs/getting-started.md

```diff
@@ -3,28 +3,28 @@
 The easiest way to get started with Octokit is to use a plain `GitHubClient`:
 
 ```csharp
-var client = new GitHubClient(new ProductHeaderValue("my-cool-app"));
+var client = new GitHubClient(new ProductHeaderValue("my-cool-app"));removed
 ```
 
-This will let you access unauthenticated GitHub APIs, but you will be subject to rate limiting (you can read more about this [here](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting)).
+This will let you access unauthenticated GitHub APIs, but you will be subject to rate limiting (you can read more about this [here](https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting)).
 
 But why do you need this `ProductHeaderValue` value?
 
-The API will reject you if you don't provide a `User-Agent` header (more details [here](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#user-agent-required)). This is also to identify applications that are accessing the API and enable GitHub to contact the application author if there are problems. So pick a name that stands out!
+The API will reject you if you don't provide a `User-Agent` header (more details [here](https://docs.github.com/rest/overview/resources-in-the-rest-api#user-agent-required)). This is also to identify applications that are accessing the API and enable GitHub to contact the application author if there are problems. So pick a name that stands out!
 
 ### Authenticated Access
 
 If you want to access private repositories or perform actions on behalf of a user, you need to pass credentials to the client.
 
-There are two options supported by the API - basic and OAuth authentication.
+There are two Authentication options supported by the API. 
```

Comment:
This isn't quite true. But that's not your fault, as GitHub's authentication options are very very complicated 🙈  I think I would correct this and just say something like this:

```suggestion
You can authenticate by passing an access token, for example a personal access token (PAT), an OAuth access token or an installation token from a GitHub App
```

Reply:
(As well as accepting this suggestion, you would need to remove the bullet pointed list below it.)